tests/tests_xml.py

```diff
@@ -246,15 +246,24 @@ def test_xml_ext_obj(self):
         self.assertEqual(arg.Description, arg2.Description)
         self.assertEqual(arg.DataType, arg2.DataType)
 
+    def test_xml_enum(self):
+        o = self.opc.nodes.objects.add_variable(2, "xmlenum", 0, varianttype=ua.VariantType.Int32, datatype=ua.ObjectIds.ApplicationType)
+        self._test_xml_var_type(o, "enum")
```

Comment:
Can you add another test for an enum based on EnumValues and not string? us ua.AttributeWriteMask (i=347) for example.